packages/gsl/gsl.1.18.2/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+name: "gsl"
+version: "1.18.2"
+maintainer: "markus.mottl@gmail.com"
+authors: [ "Olivier Andrieu <oandrieu@gmail.com>"
+           "Markus Mottl <markus.mottl@gmail.com>" ]
+license: "GPL-3+"
+homepage: "http://mmottl.github.io/gsl-ocaml"
+dev-repo: "https://github.com/mmottl/gsl-ocaml.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocamlfind" "remove" "gsl"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-bigarray"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
This camlp4 line should just be `camlp4` -- the version constraints are set from the current compiler in the `camlp4` package description.